libafl_bolts/src/tuples.rs

```diff
@@ -448,6 +456,13 @@ where
             self.1.name(index - 1)
         }
     }
+
+    fn names(&self) -> Vec<Cow<'static, str>> {
+        let first = self.0.name().to_owned();
+        let mut last = self.1.names();
+        last.push(first);
```

Comment:
This is the `names_reversed`

Reply:
Either add back a `names()` fn that calls down to this function and then `reverse()s` this vec  and rename this function to `names_reversed`, or use `.insert` instead of push to prepend (which needs a lot of shifting around elements)

Reply:
i used insert